### PR TITLE
more efficient code,  c_can.c

### PR DESCRIPTION
**Pull request by roissy**

### CODE REVIEW

**Comment by RobertCochran** [comment]
@roissy: Linus does not accept PRs from Github. You must send this patch by email to the maintainer, which in this case is `Wolfgang Grandegger <wg@grandegger.com>`, and CC the appropriate subsystem mailing lists, which in this case, happen to be: `linux-can@vger.kernel.org`, `netdev@vger.kernel.org`, `linux-kernel@vger.kernel.org`.

**Comment by ArashJavan** [comment]
@roissy : any comments why this patch??

**Comment by roissy** [comment]
I understand, thank you, I have determined that such codes more efficiently develop themselves. I will report to the necessary places. Best regards.

roissy.

File: drivers/net/can/c_can/c_can.c [hunk 1]
```diff
@@ -898,8 +898,9 @@ static int c_can_handle_state_change(struct net_device *dev,
 		cf->data[1] = (bec.txerr > bec.rxerr) ?
 			CAN_ERR_CRTL_TX_WARNING :
 			CAN_ERR_CRTL_RX_WARNING;
-		cf->data[6] = bec.txerr;
 		cf->data[7] = bec.rxerr;
+		cf->data[6] = bec.txerr;
```
Comment (fanfunny) [inline drivers/net/can/c_can/c_can.c]: 902

Comment (fanfunny) [inline drivers/net/can/c_can/c_can.c]: 902

**Pull request closed without merging**